test/unit-tests/yok.ts

```diff
@@ -601,9 +601,12 @@ describe("yok", () => {
 	});
 
 	function deleteDirectory(directory: string): boolean {
-		shelljs.rm("-rf", directory);
-		let err = shelljs.error();
-		return err === null;
+		try {
+			shelljs.rm("-rf", directory);
```

Comment:
I changed it to `shelljs.config.fatal = true` both places: https://github.com/NativeScript/nativescript-cli/pull/2364/commits/ae4937d1dada45e5e4204ed9f9c34bd78518c1a3#diff-95ad435cc796c681eb42182a4f10b038R6 and https://github.com/NativeScript/nativescript-cli/pull/2364/commits/ae4937d1dada45e5e4204ed9f9c34bd78518c1a3#diff-c095b997078fae6c5c43a57489e046b8R3